zeppelin-server/src/main/java/org/apache/zeppelin/socket/NotebookSocket.java

```diff
@@ -36,21 +40,27 @@ public NotebookSocket(Session session, Map<String, Object> headers) {
     this.session = session;
     this.headers = headers;
     this.user = StringUtils.EMPTY;
+    LOGGER.debug("NotebookSocket created for session: {}", session.getId());
   }
 
   public String getHeader(String key) {
-    return String.valueOf(headers.get(key));
+      return String.valueOf(headers.get(key));
```

Comment:
nit - please remove the two spaces. Zeppelin uses in most java classes an indentation of two spaces